swan-cern/values.yaml

```diff
@@ -171,6 +171,8 @@ swan:
         SwanKubeSpawner:
           # memory request for user pod (fraction of the limit)
           mem_request_fraction: 0.6
+        SpawnHandlersConfigs:
```

Comment:
Commit message: there will be no k8s cluster in the TN, there will be a SWAN deployment that is exposed to the TN.